frontend/packages/dev-console/src/models/knative.ts

```diff
@@ -1,5 +1,31 @@
 import { K8sKind } from '@console/internal/module/k8s';
 
+export const ConfigurationModel: K8sKind = {
```

Comment:
I've created a PR that also adds these models. We should align.
https://github.com/openshift/console/pull/1901

Reply:
sure, i'll update my PR once your goes in